frontend/src/components/NewShoot/NewShootInfrastructureDetails.vue

```diff
@@ -272,14 +273,8 @@ export default {
       }
       return validationErrors
     },
-    costObjectSettings () {
-      return getCostObjectSettings() || {}
-    },
-    costObjectSettingEnabled () {
```

Comment:
was this not used before?

Reply:
not in NewShootInfrastructureDetails, probably this is has been obsolete with the introduction of `requiresCostObjectIfEnabled`?

Reply:
maybe?

Reply:
just checked again and indeed it was missing and is required internally for requiresCostObjectIfEnabled